canvas_modules/common-canvas/src/palette/palette-flyout-content.jsx

```diff
@@ -71,41 +72,21 @@ class PaletteFlyoutContent extends React.Component {
 		const contentDivs = [];
 		for (let idx = 0; idx < categories.length; idx++) {
 			const category = categories[idx];
-			const isLastCategory = idx === categories.length - 1;
-			const nodeTypeInfos = category.node_types.map((nt) => ({ nodeType: nt, category: category }));
-			const isCategorySelected = this.isCategorySelected(category.id);
-
-			let content = null;
-			if (isCategorySelected) {
-				content = (
-					<PaletteContentList
-						key={category.label + "-nodes"}
-						show
-						category={category}
-						nodeTypeInfos={nodeTypeInfos}
-						canvasController={this.props.canvasController}
-						isPaletteOpen={this.props.isPaletteOpen}
-						isLastCategory={isLastCategory}
-						isEditingEnabled={this.props.isEditingEnabled}
-					/>);
-			}
-
 			contentDivs.push(
 				<div key={category.label + "-container"}>
 					<PaletteFlyoutContentCategory
 						key={category.id}
 						category={category}
-						isCategorySelected={isCategorySelected}
 						categorySelectedMethod={this.categorySelected}
-						itemCount={nodeTypeInfos.length}
 						canvasController={this.props.canvasController}
 						isPaletteOpen={this.props.isPaletteOpen}
+						isEditingEnabled={this.props.isEditingEnabled}
+						// isShowRanking // Uncomment this to show ranking for debuggig ranking algorithm.
```

Comment:
This is the wrong place

Reply:
I removed it from here and pasted it in the previous place, where it was in the first place !!